uint32 serial_id
float64[3] goal
---
path_info[] path